Index: Assets/nTools/PrefabPainter/Examples.meta
===================================================================
--- Assets/nTools/PrefabPainter/Examples.meta	(revision 972cdd7f93cd9ff9572d735dafde1157190cf07d)
+++ Assets/nTools/PrefabPainter/Examples.meta	(revision 972cdd7f93cd9ff9572d735dafde1157190cf07d)
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
-guid: dd82b5ea95706f340b552c424382c7dd
-folderAsset: yes
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_06_01_2021_15_30__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_06_01_2021_15_30__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_02_01_2021_15_46__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_02_01_2021_15_46__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_22_12_2020_11_05__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_22_12_2020_11_05__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_07_01_2021_18_58__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_07_01_2021_18_58__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_06_01_2021_15_29__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_06_01_2021_15_29__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_01_01_2021_17_50__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_01_01_2021_17_50__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_01_01_2021_18_38__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Merge_at_01_01_2021_18_38__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_02_01_2021_15_46__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Checkout_at_02_01_2021_15_46__Default_Changelist_.xml
diff --git .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Update_at_07_01_2021_19_01__Default_Changelist_.xml .idea/.idea.Swarm/.idea/shelf/Uncommitted_changes_before_Update_at_07_01_2021_19_01__Default_Changelist_.xml
